source/_sass/page.scss

```diff
@@ -1,333 +1,330 @@
-		display: block;
-		list-style-type: none;
-		border-bottom: 1px solid lighten($black,80);
-		border-bottom: 1px solid fade($black,10);
-		@include font-rem(16);
-		&:last-child {
-			border-bottom: 0 solid transparent;
-		}
-		@media #{$small} {
-			display: inline;
-			margin-right: 25px;
-			white-space: nowrap;
-			border-bottom: 0 solid transparent;
-		}
-		a {
-			display: block;
-			padding: 10px 0;
-			decoration: none;
-			border-bottom: 0 solid transparent;
-			@include transition(all .2s);
-			@media #{$small} {
-				display: inline;
-				padding: 0;
-			}
-		}
-	}
+  @include grid(12,10);
+  @include prefix(12,1);
+  @include suffix(12,1);
+  margin-bottom: 1em;
+  float: none;
+  display: block;
+  @media #{$small} {
+    @include grid(12,9);
+    @include prefix(12,0);
+    @include suffix(12,0);
+  }
+  @media #{$x-large} {
+    @include grid(12,8);
+  }
+  ul {
+    margin: 0;
+    padding: 0;
+    clear: both;
+    list-style-type: none;
+  }
+  li {
+    display: block;
+    list-style-type: none;
+    border-bottom: 1px solid lighten($black,80);
+    border-bottom: 1px solid fade($black,10);
+    @include font-rem(16);
+    &:last-child {
+      border-bottom: 0 solid transparent;
+    }
+    @media #{$small} {
+      display: inline;
+      margin-right: 25px;
+      white-space: nowrap;
+      border-bottom: 0 solid transparent;
+    }
+    a {
+      display: block;
+      padding: 10px 0;
+      decoration: none;
+      border-bottom: 0 solid transparent;
+      @include transition(all .2s);
+      @media #{$small} {
+        display: inline;
+        padding: 0;
+      }
+    }
+  }
 }
 // Navicon lines
 $button-size: 1.5rem;
 @mixin navicon-line() {
-	display: inline-block;
-	width: $button-size;
-	height: $button-size/7;
-	// line color
-	background: $white;
-	border-radius: $button-size/14;
-	transition: .3s;
+  display: inline-block;
+  width: $button-size;
+  height: $button-size/7;
+  // line color
+  background: $white;
+  border-radius: $button-size/14;
+  transition: .3s;
 }
 .navicon-lines-button {
-	padding: $button-size/4 $button-size/2;
-	transition: .3s;
-	cursor: pointer;
-	user-select: none;
-	border-radius: $button-size/7;
+  padding: $button-size/4 $button-size/2;
+  transition: .3s;
+  cursor: pointer;
+  user-select: none;
+  border-radius: $button-size/7;
 }
 .navicon-lines-button:hover {
-	opacity: 1;
+  opacity: 1;
 }
 .navicon-lines-button:active {
-	transition: 0;
+  transition: 0;
 }
 .navicon-lines {
-	margin-right: 10px;
-	margin-bottom: $button-size/5;
-	// create middle line
-	@include navicon-line;
-	position: relative;
-	// create the upper and lower lines as pseudo-elements of the middle line
-	&:before,
-	&:after {
-		@include navicon-line;
-		position: absolute;
-		left: 0;
-		content: '';
-		-webkit-transform-origin: $button-size/14 center;
-		transform-origin: $button-size/14 center;
-	}
-	&:before { top: $button-size/4; }
-	&:after { top: -$button-size/4; }
+  margin-right: 10px;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets